tools_considered_and_the_screening_decisions.md

```diff
@@ -0,0 +1,178 @@
+- [González et al. 2021](https://doi.org/10.11144/Javeriana.10554.58492): *Tools from Table 1 with label: 'selección de estudios'*: Buhos, SRDB.PRO, SLuRp, DistillerSR, JBI-SUMARI, Nested-Knowledge, StArt, TaskExchange, CADIMA, EPPI-Reviewer, Parsifal, ReLiS, SESRA, Review Manager (RevMan), SLR-Tool, Systematic Review Accelerator, Thoth, ASReview, Covidence, PICO Portal, Rayyan, Health Assessment Workspace Collaborative (HAWC), Lingo3G: text document clustering engine, MeSHSIM, SLR.qub, SyRF: Systematic Review Facility. *Tools from Table 2*: Rayyan, Covidence, DRAGON, Abstrackr, Colandr, EPPI-Reviewer
+- [Jimenez et al. 2022](https://doi.org/10.1186/s12874-022-01805-4): Active_learning_document_screening, Active-learning-for-systematic-review, DAE-FF, InclusionCriteria, Machine Learning Functions, PubmedClassifier, Rules_cochranereviews, SLR_SearchStrings, ASReview, ASReview-covid, FASTREAD, GAPscreener, Pvtopic, RapidMiner, Rax, SWIFT-Review, Revtools, Abstrackr, Cochrane RCT Classifier, Colandr, Concept Encoder, DistillerSR, DoCTER, EPPI-Reviewer, IRIS.AI, PICO Portal, Rayyan, Research Screener, RobotAnalyst, RobotSearch, Screen4Me, SRA, SWIFT-Active Screener, SyRF, Sysrev
+- [Khalil et al 2022](https://doi.org/10.1016/j.jclinepi.2021.12.005): Rayyan, EPPI – reviewer, Abstrackr, SRA helper, LibSVM classifier, Bibot, DistelleR, Active Screener, RobotAnalyst, Swift-Review
+- [Pellegrini et al. 2021](https://doi.org/10.36253/form-11343): ASReview, Covidence, EPPI-Reviewer, Rayyan
+- [Robledo et al. 2021](https://doi.org/10.1080/00472778.2021.1955125): Rayyan, SWIFT-Review, ASReview
+- [Scott et al. 2021](https://doi.org/10.1016/j.jclinepi.2021.06.030): *Tools related to the screening phase*: Covidence, RevMan, Rayyan, JBI-SUMARI, DistillerSR, EPPI-Reviewer, Systematic Review Accelerator (screenatron), SRA-Helper (Endnote Helper)
+- [Tsou et al. 2021](https://doi.org/10.1186/s13643-020-01324-7): Abstrackr, EPPI-Reviewer
+- [van_de_Schoot et al. 2021](https://doi.org/10.1038/s42256-020-00287-7): Abstrackr, ASReview, Colandr, FASTREAD, Rayyan, RobotAnalyst
+- [Wagner et al. 2021](https://doi.org/10.1177/02683962211048201): ASReview, ADIT approach
+- [Wang and Lo 2021](https://doi.org/10.1093/bib/bbaa296): ASReview, Rayyan, RobotAnalyst, RobotReviewer, Trialstreamer
+- [Xuan et al. 2021](http://www.hxyx.com/article/54495.html): Abstrackr, EPPI-reviewer, Rayyan, ASReview, DoCTER, Colandr, SWIFT-Active Screener, GAPscreener
+
+
+## Search 2
+
+The Systematic Review Toolbox is an online catalogue of tools that can support researchers in the different stages of performing a systematic review. We searched this toolbox for screening tools, with the following search criteria:
+
+Search in the [SR Toolbox](http://systematicreviewtools.com/): 
+`Review family: "any" | Cost: "any" | Stages of the review: "Screening" | 44 tools have been found.` 
```

Comment:
and selected "Software" instead of "Guidance"

Reply:
sorry, maybe I did not understand your comment